crates/taplo/src/formatter/mod.rs

```diff
@@ -361,14 +363,14 @@ struct FormattedEntry {
     key: String,
     // This field is used to cache the "cleaned" version of the key and should only
     // be accessed through the `cleaned_key` helpers method.
-    private_cleaned_key: OnceCell<Vec<String>>,
+    cleaned_key: OnceCell<Vec<String>>,
```

Comment:
I think the idea was to also make the comment a documentation by using `///` instead of `//` (even though it's a private field; it's possible to generate documentation using `--document-private-items` and rust-analyzer always uses them).